### PR TITLE
`azurerm_cosmosdb_cassandra_datacenter` - add availability zones flag

### PR DESCRIPTION
**Pull request by TheovanKraay**

### CODE REVIEW

**Review by WodansSon — APPROVED** [review]
@TheovanKraay, thanks for the PR! LGTM! 🚀

File: internal/services/cosmos/cosmosdb_cassandra_datacenter_resource.go [hunk 1]
```diff
@@ -81,6 +81,11 @@ func resourceCassandraDatacenter() *pluginsdk.Resource {
 				Optional:     true,
 				ValidateFunc: validation.IntBetween(1, 10),
 			},
+			"availability_zones": {
```
Comment (katbyte) [inline internal/services/cosmos/cosmosdb_cassandra_datacenter_resource.go:84]: could we make this
```suggestion
			"availability_zones_enabled": {
```

**Comment by WodansSon** [comment]
@katbyte due to time zone shifts... can we merge this PR and I will open another one to change the attribute name?

**Review by katbyte — APPROVED** [review]
works for me @WodansSon !

**Comment by katbyte** [comment]
new PR #14237

**Pull request merged**